packages/scaffold-ui/src/views/w3m-approve-transaction-view/index.ts

```diff
@@ -71,38 +77,27 @@ export class W3mApproveTransactionView extends LitElement {
         this.iframe.style.bottom = 'unset'
       }
       this.ready = true
+      this.onShowIframe()
     })
     this.bodyObserver.observe(window.document.body)
   }
 
   // -- Render -------------------------------------------- //
   public override render() {
-    if (this.ready) {
-      this.onShowIframe()
-    }
-
     return html`<div data-ready=${this.ready}></div>`
   }
 
   // -- Private ------------------------------------------- //
   private onShowIframe() {
     const isMobile = window.innerWidth <= 430
-    this.iframe.animate(
-      [
-        { opacity: 0, transform: isMobile ? 'translateY(50px)' : 'scale(.95)' },
-        { opacity: 1, transform: isMobile ? 'translateY(0)' : 'scale(1)' }
-      ],
-      { duration: 200, easing: 'ease', fill: 'forwards' }
-    )
+    this.iframe.style.animation = isMobile
+      ? 'w3m-iframe-zoom-in-mobile 200ms var(--wui-ease-out-power-2)'
+      : 'w3m-iframe-zoom-in 200ms var(--wui-ease-out-power-2)'
```

Comment:
```suggestion
    this.iframe.style.animation = `w3m-iframe-zoom-in${isMobile ? '-mobile' : ''} 200ms var(--wui-ease-out-power-2)`
```

Reply:
(just a nit)

Reply:
Sounds good idea, I've tried but when format the file, it's looking less readable IMO:

```jsx
this.iframe.style.animation = `w3m-iframe-zoom-in${
  isMobile ? '-mobile' : ''
} 200ms var(--wui-ease-out-power-2)`
```